contents/blog/new-vp-nailing-funnels/new-vp-nailing-funnels.md

```diff
@@ -0,0 +1,80 @@
+
+**Competitor research**
+
+When you’re new to a market with existing competitors it’s important to understand who they are and what’s unique about their products. I analyzed the marketing and documentation of a number of our competitors to understand our potential gaps and advantages. It’s key not to get hung up on features our competitors have which we don’t - we must focus on building the best solution to the problems our customers have, which might look different to what our competitors offer.
+
+**Analytics**
+
+PostHog is a product analytics tool, so I was able to get set-up and analyse the behavior of our users within a few hours. I focused on breaking down how our existing product is used across different dimensions to understand which of our capabilities are most valuable to our existing customers and have the biggest growth opportunities.
+
+### Product
+
+**Mission**
+
+Posthog’s mission to `"Increase the number of successful products in the world"` was a big reason I joined the company, and that phrase provided context to building out a long-term direction.
+
+Dogfooding
+
+Every good product manager is one of the world’s top users of their product. I spent over 12 hours in my first week using the product inside and out, identifying bugs, drawbacks and opportunities for the product along the way.
+
+**Tasks and feature requests**
+
+To gather context on how the team prioritizes and what they’re currently working on, I spent a number of hours reading through open GitHub issues. This gave me insight into what our community is expecting and experiencing when they use PostHog.
+
+### People
+
+**Intro interviews and insights**
+
+I attempted to meet with every single person in the company in my first week - a few meetings spilled over into the second week. I introduced myself and got to learn about them, their history, and what they’re passionate about. I asked everyone the same three questions:    
+
+* In your opinion, what's the long term direction of PostHog?
+* What do you think is the biggest blocker to our success?
+* What can I do to help you become more successful?
+
+The answers to these questions were really enlightening. It was immediately clear that the company had been quite reactive and that a proactive strategy would be welcomed. I also got a great understanding of what the future of PostHog might look like and what I can do to help each individual succeed.
+
+### Insights
+
+Based on these interviews, I aggregated the key themes as a collection of insights to share with the team. This also gave me the opportunity to identify and share a number immediate actions I would be taking to help push the company forward.
```

Comment:
Recommend changing `share a number immediate actions` to `share a number of immediate actions`

Reply:
Amended